hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/realtime/HoodieParquetRealtimeInputFormat.java

```diff
@@ -66,6 +90,138 @@
     return HoodieRealtimeInputFormatUtils.getRealtimeSplits(job, fileSplits);
   }
 
+  /**
+   * keep the logical of mor_incr_view as same as spark datasource.
+   * to do: unify the incremental view code between hive/spark-sql and spark datasource
```

Comment:
Can we unify the logic in this PR ?

Reply:
the logic has already be unified，now the logic of hive incremental and spark incremental is  same。     but to do unify the code  has  a  little difficult。  may be can do it in another pr。